actions/ostree_deploy_action.go

```diff
@@ -1,3 +1,46 @@
+/*
+OstreeDeploy Action
+
+Put the selected OSTree branch to target target image.
```

Comment:
Deploy the ostree branch ..

Reply:
target target ? :)
Also target image is weird, it's not explicitely targetting. to the image might be better then using target image

Reply:
should be *the* target, but now it is just the image ;)